### DISCUSSION TITLE
while resolving dids this warning occurs in veramo for react-native

### DISCUSSION THREAD

**Discussion by Musa00789** [decentralized-identity/veramo#1093] [General]
Possible Unhandled Promise Rejection (id: 1):
Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.7.2)

**Comment by nickreynolds** [comment]
can you post the DID you are trying to resolve? and your agent config?

if you post a link to a github repo we could attempt to reproduce the error
